lab_1_classify_by_unigrams/main.py

```diff
@@ -2,32 +2,34 @@
 Lab 1
 Language detection
 """
-
-
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+    if not isinstance(text, str):
+        return None
+    else:
+        text = text.lower()
+        cleaned_text = ""
+        for symbol in text:
+            if symbol.isalpha() and symbol != " ":
+                cleaned_text += symbol
+        tokens = list(cleaned_text)
+        return tokens
 
 
 def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
-    """
-    Calculates frequencies of given tokens
-    :param tokens: a list of tokens
-    :return: a dictionary with frequencies
-    """
+    freqs = {}
+    for token in tokens:
+        if token in freqs:
```

Comment:
Lets set 0 if token is not in freqs and do +1 always

Reply:
I did this but I got errors